server/src/main/java/org/opentosca/toscana/plugins/cloudformation/visitor/CheckModelNodeVisitor.java

```diff
@@ -55,19 +58,34 @@ public void visit(Compute node) {
             }
         }
     }
+    
+    @Override
+    public void visit(Database node) {
+        // noop
+    }
 
     @Override
     public void visit(MysqlDatabase node) {
         // noop
     }
-
+    
+    @Override
+    public void visit(Dbms dbms) {
+        // noop
+    }
+    
     @Override
     public void visit(MysqlDbms node) {
         // noop
     }
-
+    
```

Comment:
Please run a reformat with the proper settings to resolve such issues. (empty lines should not contain spaces)